app-backend/db/src/main/scala/CampaignDao.scala

```diff
@@ -34,10 +34,16 @@ object CampaignDao extends Dao[Campaign] with ObjectPermissions[Campaign] {
     "children_count",
     "project_statuses",
     "is_active",
-    "resource_link"
+    "resource_link",
+    "task_status_summary"
   )
 
-  def selectF: Fragment = fr"SELECT " ++ selectFieldsF ++ fr" FROM " ++ tableF
+  def selectF: Fragment =
+    fr"SELECT " ++ selectFieldsF ++ Fragment.const(
+      ", COALESCE(image_count, 0) as image_count"
```

Comment:
I couldn't put the image count in the select fields without causing some more downstream issues - the select fields are used pretty heavily in copying/cloning and for image counts this value will always be calculated on the fly